README.md

```diff
@@ -92,12 +92,34 @@ cp $GOPATH/bin/dcr* bin/
 yarn dev
 ```
 
-### Note about developing with testnet
+## Setting up your development environment
+The following steps will help you configure your decrediton development environment and reduce future startup times.
 
-The first time you run with testnet, you may get a "Failed to connect wallet by deadline error".  If so, make sure you change your config.json:
-```bash
-"network": "testnet",
-```
+### Wallet
+When you launch decrediton, you will be prompted to select a wallet to use. Select your wallet or create a new one using the in-app wizard. Be sure to save your seed and make your password memorable.
+
+### Decred Node
+It will be helpful to you to run the Decred node in a separate process and simply attach to it between decrediton restarts. In order to see the advanced daemon configuration options you open your ```config.json``` and set the ```daemon_start_advanced``` flag to ```true``` as follows:
+
+```"daemon_start_advanced": true,```
+
+Note: Your config.json file is located in the following directory(s)
+
+Windows - ```~/.config/decrediton/config.json``` 
```

Comment:
This path is not correct.  C:\Users\<your-username>\AppData\Local\Decrediton\config.json